Add hormone-levels-female-puberty

Aquaticat opened the pull request:
有一些不清楚如何翻译的地方，请提建议：

1. trackHash 是什么？

2. keywords 应写哪些？

3. [wiki-aaaaaaaa] 这个写法是什么？

应该还有很多其它翻译得不对或者不准确的地方。请指导。

Signed-off-by: Aquaticat <an@aquati.cat>

Aquaticat committed:
Create hormone-levels-during-normal-puberty-in-cisgender-girls

有一些不清楚如何翻译的地方，请提建议：

1. trackHash 是什么？

2. keywords 应写哪些？

3. [wiki-aaaaaaaa] 这个写法是什么？

应该还有很多其它翻译得不对或者不准确的地方。请指导。

Signed-off-by: Aquaticat <an@aquati.cat>

Bersella-AI commented:
1. `trackHash` 和 `[wiki-1a2b3c4d]` 等标识需配合一个追踪 TFsci 原站文章更新的 repo 来使用，但该 repo 仅对 MtF.wiki 内部可见。`trackHash` 可以留空。
2. `keywords` 意为“关键词”，需要填上简要概括文章内容的几个词语。可以留空。

Aquaticat committed:
Correct hormone-levels-during-normal-puberty-in-cisgender-girls according to Bersella AI's comments

这个修改是去掉了您给的注释的。有注释的版本我也留了。

Signed-off-by: Aquaticat <an@aquati.cat>

Aquaticat committed:
Correct file extension to hormone-levels-during-normal-puberty-in-cisgender-girls.md

Signed-off-by: Aquaticat <an@aquati.cat>

Aquaticat committed:
Rename hormone-levels-during-normal-puberty-in-cisgender-girls.md to hormone-levels-female-puberty.md

Signed-off-by: Aquaticat <an@aquati.cat>

septs commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,351 @@
+  - Bersella AI
+tags:
+  - 雌激素
+  - 孕激素
+  - 孕激素
```
```suggestion
  - 孕激素
```

septs commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,351 @@
+updated: 2021-08-06
+translated: 2023-02-06
+translators:
+  - Aquaticat
```
https://github.com/mtf-wiki/transfeminine-science/blob/6905175603112868e55454981a15602adcc3f0f9/content/about.md?plain=1#L105-L112

在 about 添加关于自己的描述

Aquaticat replied:
已添加。

Aquaticat committed:
Fix front matter duplication in content/articles/hormone-levels-female-puberty.md

Co-authored-by: septs <github@septs.pw>
Signed-off-by: Aquaticat <an@aquati.cat>

Aquaticat committed:
Added description of Aquaticat in about.md as requested from https://github.com/mtf-wiki/transfeminine-science/pull/7/files#r1101068298

在 about 添加了关于自己的描述。

Signed-off-by: Aquaticat <an@aquati.cat>

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,350 @@
+
+| 生命阶段或年龄     | 范围 (ng/mL) |
+|-------------|------------|
+| 5–9 years   | 	≤0.6      |
+| 10–13 years | 	≤10.2     |
+| 14–17 years | 	≤11.9     |
```
还是有遗漏的地方……

Aquaticat replied:
好的。

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,350 @@
+  The Journal of Steroid Biochemistry and Molecular
+  Biology, 183, 116–124. [DOI:10.1016/j.jsbmb.2018.06.005]
+
+    * Has estrone sulfate levels in the different Tanner breast stages.
+      在不同的Tanner 乳房发育阶段的雌酮水平。
```
这行字其实是原作者的注解，不是被引用的部分。只有引用文献要保留原文。

Aquaticat replied:
好的。

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,350 @@
+  发育期前女孩的荷尔蒙变化
+  The Journal of Clinical Endocrinology & Metabolism, 99(10), 3829–3835. [DOI:10.1210/jc.2013-4528]
+
+* Ankarberg-Lindgren, C., Dahlgren, J., & Andersson, M. X. (2018). High-sensitivity quantification of serum androstenedione, testosterone, dihydrotestosterone, estrone and estradiol by gas chromatography–tandem mass spectrometry with sex-and puberty-specific reference intervals.
+  按照性别和发育期定下了不同的血清雄烯二酮、睾酮、双氢睾酮、雌酮和雌二醇水平的参考范围。
+  以气相色谱-串联质谱法分别对不同性别和是否处于发育期的人的血清雄烯二酮、睾酮、双氢睾酮、雌酮和雌二醇水平进行高灵敏度的定量分析。
```
第 163 行似乎是早期的翻译，除去。

Aquaticat replied:
此处既要表达”对不同人群定下不同的各激素水平参考范围“，又要表达“以...法，对各激素水平进行高灵敏度的定量分析”。所以我当时认为写成两句话更通顺。

Bersella-AI replied:
但需注意，译文*只需* 保留一个版本。给读者*做选择* 是很不负责任的，会造成严重困惑。此处 `-specific` 后缀没有“按照……定下”这样的**主观意识**，只是用于按照**客观事实**区分人群类型。

Aquaticat replied:
好的。

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,350 @@
+
+*表格 2*：以液相色谱-质谱联用法（LC-MS）或质谱法（MS）测定，顺性别女孩和成年女性的雌二醇和雌酮的水平：
+
+|                 | 雌二醇           | 雌酮           |
+|-----------------|---------------|--------------|
+| 生命阶段或年龄         | 参考范围 (pg/mL)	 | 参考范围 (pg/mL) |
```
没必要完全遵循原站的格式和排版。可以这么处理：
```
| 生命阶段或年龄 | 雌二醇参考范围 | 雌酮参考范围 |
|-|-|-|
| …… | …… | …… |
```

Aquaticat replied:
好的。

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,350 @@
+  The Journal of Clinical Endocrinology & Metabolism, 95(1),
+  82–92. [DOI:10.1210/jc.2009-1140]
+
+* Biro, F. M., Pinney, S. M., Huang, B., Baker, E. R., Walt Chandler, D., & Dorn, L. D. (2014). Hormone changes in peripubertal girls.
+  发育期前女孩的荷尔蒙变化
```
前缀 peri-\* 是“某个时期当中”的意思。“某个时期前”用 pre-\* ；“某个时期后”用 post-\* 。

Aquaticat replied:
好的。

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,350 @@
+| 怀孕第二阶段      | 19.5–82.5  |
+| 怀孕第三阶段      | 65.0–229.0 |
+
+少年男子（1 - 10 岁和发育期）的孕酮水平 <0.10–1.1 ng/mL。成年男子的孕酮水平 0.13–0.97 ng/mL。
```
1 ~ 10 岁到发育期，可统称“男孩”。现代中文一般不用“少年男子”称呼，而用“少年”。

Aquaticat replied:
此处是遗漏。已修改。

Aquaticat committed:
Fix hormone-levels-female-puberty.md according to comments.

URL of requested changes:
https://github.com/mtf-wiki/transfeminine-science/pull/7/files/1d0e0f480f4ef724de755e66c2617df84c25d630

Signed-off-by: Aquaticat <an@aquati.cat>

septs commented:
@Bersella-AI 可以考虑以 track 的底稿为基础吧以翻译的文本替换其中，以提高效率

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,348 @@
+用于检测女性是否开始青春期的超声波乳腺扫描和内分泌化验的激素参考值。 The Journal of Clinical Endocrinology & Metabolism,
+105(12), dgaa679. [DOI:10.1210/clinem/dgaa679]：
+
+*表格 4*：以 LC-MS/MS 测得的顺性别女孩雌二醇和雌酮的水平：
+
+|             |       | 雌二醇        |           | 雌酮         |           |
+|-------------|-------|------------|-----------|------------|-----------|
+| 发育阶段        | n     | 中位数（pg/mL） | 范围（pg/mL） | 中位数（pg/mL） | 范围（pg/mL） |
+| Tanner 阶段 1 | 260   | 1.3        | 5.0-20    | 13         | 4.0-29    |
+| Tanner 阶段 2 | 69    | 6.5        | 10-24     | 21         | 10-33     |
+| Tanner 阶段 3 | 63-66 | 25         | 7.0-60    | 30         | 15-43     |
+| Tanner 阶段 4 | 73-78 | 48         | 21-85     | 36         | 16-77     |
+| Tanner 阶段 5 | 76-81 | 41         | 34-170    | 61         | 29-105    |
```
表四数据与原文不符，请更正。

Aquaticat replied:
好的

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,348 @@
+  The Journal of Steroid Biochemistry and Molecular
+  Biology, 183, 116–124. [DOI:10.1016/j.jsbmb.2018.06.005]
+
+    * 在不同的 Tanner 乳房发育阶段的雌酮水平。
```
需要重新翻译：原文是指（上述论文）有这样的数据，而且 estrone sulfate（硫酸雌酮）不是雌酮，是雌酮的一种代谢物。

Aquaticat replied:
好的。

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,348 @@
+
+    * 在不同的 Tanner 乳房发育阶段的雌酮水平。
+
+* Bae, Y. J., Zeidler, R., Baber, R., Vogel, M., Wirkner, K., Loeffler, M., … & Kratzsch, J. (2019). Reference intervals of nine steroid hormones over the life-span analyzed by LC-MS/MS: Effect of age, gender, puberty, and oral contraceptives.
+  探讨年龄，性别，发育期，和口服式避孕药对以 LC-MS/MS 测得的九种类固醇荷尔蒙的参考范围的影响。
```
这样的语序有点令人不明所以。“以 LC-MS/MS 测定”可以作为附注，加上括号置于句末。此外，“荷尔蒙”是较过时的说法，用“激素”即可。
我对此标题的释义是：“一生里人体九种类固醇激素的参考范围（以 LC-MS/MS 测定），以及年龄、性别、青春期和口服避孕药对其的影响。”

Aquaticat replied:
好的。还是觉得不通顺，不过也想不到更好的表达方式了。

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,348 @@
+
+### Quest Diagnostics (2007) {#quest-diagnostics-2007}
+
+Fisher, D. A., Quest Diagnostics, Nichols Institute (2007). The Quest Diagnostics Manual: Endocrinology: Test Selection
+and Interpretation, 4th Edition.
+
+Fisher, D. A., Quest Diagnostics, Nichols Institute (2007). 奎斯特诊断公司操作手册：内分泌学测试选择与诠释，第四版。[[PDF](http://docplayer.net/86293048-The-quest-diagnostics-manual-endocrinology-test-selection-and-interpretation-fourth-edition.html)] [[Alt PDF](https://studylib.net/doc/8074566/test-selection-and-interpretation-the-quest-diagnostics-m)] [[Alt](https://books.google.com/books?id=BBLRUI4aHhkC&pg=PA1982)]:
```
这两行应合并。

Aquaticat replied:
好的

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,348 @@
+| 成人黄体期       | ≤31.4      |
+| 停经后         | ≤0.2       |
+
+5 - 17 岁青少年男孩的孕酮水平 ≤0.3 ng/mL。成年男性的孕酮水平 ≤1.2 ng/mL。
```
二者的数据弄反了。

Aquaticat replied:
好的

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,348 @@
+
+Esoterix/LabCorp. (2020). 激素水平的期望值和国际单位的换算表。[[PDF](https://web.archive.org/web/20200330030510/https://www.esoterix.com/sites/default/files/L5167-0320-17.pdf)] [[Alt](https://web.archive.org/web/20180814132700/https://ltd.aruplab.com/Tests/Pub/2008509)] [[Alt](https://doi.org/10.1007/978-3-319-18371-8_16)]:
+
+*表格 8*：以 HPLC-MS/MS 测得的顺性别女性睾酮的水平：
+
+| 生命阶段         | 年龄         | 平均值 (ng/dL) | 范围 (ng/dL) |
```
表八至表十一的“年龄”应加上单位“岁”。

Aquaticat replied:
好的

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,348 @@
+| Tanner 阶段 3	 | 10.0–14.4  | 	97	        | 50–170     |
+| Tanner 阶段 4	 | 10.7–15.6  | 	105	       | 47–208     |
+| Tanner 阶段 5  | 	11.8–18.6 | 137         | 	50–224    |
+| 成人停经前期	      | >18.0	     | ?           | 	28–230    |
```
表十这行的年龄并非“大于 18”，而是 18-40。

Aquaticat replied:
好的

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,348 @@
+| Tanner 阶段 3	 | 10.0–14.4  | 	88         | 	32–226    |
+| Tanner 阶段 4	 | 10.7–15.6  | 	120	       | 58–260     |
+| Tanner 阶段 5  | 	11.8–18.6 | 	148	       | 44–248     |
+| 成人停经前期	      | >18.0	     | ?           | 	17–372    |
```
表十一这行的年龄并非“大于 18”，而是 18-50。

Aquaticat replied:
好的

Bersella-AI commented on content/articles/hormone-levels-female-puberty.md:
```diff
@@ -0,0 +1,348 @@
+### 其它相关研究 {#other-studies-2}
+
+* Kushnir, M. M., Rockwood, A. L., Roberts, W. L., Pattison, E. G., Bunker, A. M., Fitzgerald, R. L., & Meikle, A. W. (
+  2006). Performance characteristics of a novel tandem mass spectrometry assay for serum testosterone.
+  一种新型的血清睾丸激素串联质谱分析法的性能和特点。
```
对同一名词、术语的翻译应*至少* 在同一文章内一致。不能这边用“睾丸激素”，那边用“睾酮”。

Aquaticat replied:
好的

Aquaticat committed:
Fix hormone-levels-female-puberty.md according to comments.

https://github.com/mtf-wiki/transfeminine-science/pull/7/files/09755b3b65eef8eec4717bcb2b5932473660b0e7

Signed-off-by: Aquaticat <an@aquati.cat>

Bersella-AI reviewed (approved):
校对得差不多了。为了和原站文章、以及本站译文的高标准相契合，这篇译文前前后后一共指出了超过 50 项问题，我花的时间相当于**把全文核对五六遍**；但这种工作是值得的：**这会为今后其他有意参与的伙伴提供一个规范化、高标准的指引**。我会择机以此 PR 为例发表《贡献指南》。
另外提醒所有意欲参与的伙伴：各位的贡献质量如何、是否能够节约我的时间和精力，都不重要——事实上我对译文的基本要求只有“**如实传达**”（项目主页原文）；重要的是**这对你自己而言是否值得去付出时间和精力**。光有热爱是不够的，还需各位斟酌一二。
